__tests__/get_all_pixels.spec.ts

```diff
@@ -0,0 +1,78 @@
+import request from 'supertest';
+import { BASE_URL, TOKEN } from '../helpers/constant'
+import { generateRandomString, generateRandomPixelType } from '../helpers/common'
+import { createPixel, deleteManyPixels } from '../helpers/api.helper'
+import { DEFAULT_TIMEOUT } from '../helpers/constant'
+
+describe('GET /api/pixels', function () {
+    var pixel_id: string[] = []
+
+    test('PIX-001 Verify that user can get list of total pixels', async function () {
+        const record_size = 2;
+        // create new pixels for testing
+        for (let i = 0; i < record_size; i++) {
+            let randomType = generateRandomPixelType();
+            let randomName = generateRandomString();
+            let randomTag = generateRandomString();
+            const newPixel = await createPixel(randomType, randomName, randomTag);
+            // add id into array for deleting later
+            pixel_id.push(newPixel.body.id);
+        };
+        // get list of pixels
+        const response = await request(BASE_URL)
+            .get("/api/pixels")
+            .set('Authorization', `${TOKEN}`);
+        const data = response.body.data;
+        expect(response.status).toEqual(200);
+        expect(response.body.error).toEqual(0);
+        expect(data.result).toEqual(record_size);
+        expect(data.perpage).toEqual(15);
+        expect(data.currentpage).toEqual(1);
+        expect(data.nextpage).toEqual(null);
```

Comment:
No needs to verify value of "record_size", "nextpage" and "maxpage". This should be verify a different test cases. For scope of this test case, just verifying body contains these fields.

Reply:
fixed